src/extension.ts

```diff
@@ -144,56 +147,19 @@ function registerOpenCsSettingsCommand(context: vscode.ExtensionContext) {
  *
  */
 function addReviewListeners(context: vscode.ExtensionContext) {
-  // This provides the initial diagnostics when a file is opened.
-  context.subscriptions.push(
-    vscode.workspace.onDidOpenTextDocument((document: vscode.TextDocument) => {
-      CsDiagnostics.review(document);
-    })
-  );
-
-  // Close document listener for cancelling reviews and refactoring requests
-  context.subscriptions.push(
-    vscode.workspace.onDidCloseTextDocument((document: vscode.TextDocument) => {
-      Reviewer.instance.abort(document);
```

Comment:
Aborting reviews after closing a tab is no longer desired, since now it depends on Git changes, not UI